.changeset/rich-actors-speak.md

```diff
@@ -0,0 +1,14 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`AppFrame`:
+- Modified sticky/fixed position to turn off when viewport height is under 480px in height
+- Refactored styles to make `AppFrame` responsible for sticky/fixed layout of `SideNav` and `AppHeader`
+
+`AppHeader`:
+- Styled inoperable actions as disabled (which occurs when the `SideNav` is expanded in mobile view)
+
+`SideNav`:
+- Removed need for `withAppHeader` option when used together with AppHeader
+- Removed the `withAppHeader` option as it is no longer needed.
```

Comment:
[todo] Are these two lines duplicated somehow?

Reply:
So first I refactored to remove the need for the `withAppHeader` argument and then I actually removed the actual argument from the code. That's what I was trying to express but I agree that's confusing so I will edit, thanks.